src/components/ProfileGrades.tsx

```diff
@@ -0,0 +1,97 @@
+import Chart from "react-apexcharts"
+import { useState } from "react"
+
+export interface GradeDistribution {
+  name: string;
+  series: ApexAxisChartSeries;
+}
+
+import { AiFillCaretLeft, AiFillCaretRight } from "react-icons/ai"
+
+const gradeChartOptions = {
```

Comment:
There sure is a lot here! Good formatting putting things on new lines, but maybe this stuff belongs in a separate config TypeScript file that you can then import this const var from?